vcpkg/get_started/get-started-linux.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Install and use packages on Linux
+description: Tutorial guides the user through the process of installing and using packages on Linux with vcpkg.
+author: JavierMatosD
+ms.date: 08/23/2023
+ms.author: javiermat
+ms.prod: vcpkg
+---
+# Install and use packages on Linux
+
+## Description
+
+This tutorial guides you through setting up a new C++ "Hello World" project using the `fmt` library. You'll manage dependencies, build the application, and eventually run your project, all using vcpkg. This tutorial assumes you have vcpkg setup and ready to use. If not, check out our initial setup [tutorial](setup-vcpkg.md).
+
+### Prerequisites
+
+For this tutorial, you'll need to have `git`, `bash`, `CMake`, and a C++ compiler installed.
+
+## Setup the project
+
+Create a directory for your new project next to your vcpkg clone and navigate into it:
+
+```bash
+mkdir helloworld && cd helloworld
+```
+
+Initiate a new project within this directory, creating the manifest (`vcpkg.json`) and default configuration files (`vcpkg-configuration.json`). From within the `helloworld` directory, run:
+
+```bash
+..\vcpkg\vcpkg new --name helloworld --version 1.0
+```
+
+>[!NOTE]
+>Ensure the project directory name matches exactly with the name you give to the `vcpkg new` command. This ensures the manifest files land in the correct directory.
```

Comment:
Sorry, my note is not descriptive enough. The problem isn't with the `vpkg new` command. This just spits out the manifest in the cwd. The problem is when you try to use commands like `vpkg add port foo`. For example, if I name my directory "helloworld", and run: `vcpkg new --name HelloWorld --version 1.0`, it will spit out a manifest in the cwd which is "helloworld". However, if I try to run `vcpkg add port fmt` then you get the following error:

```
error: while loading C:\dev\helloworld\vcpkg.json:
$.name (an identifier): "helloWorld" is not a valid identifier. Identifiers must be lowercase alphanumeric+hypens and not reserved (see https://learn.microsoft.com/vcpkg/users/manifests for more information)See https://learn.microsoft.com/vcpkg/users/manifests for more information.
```

The error message is not very descriptive of the problem.